saiku-ui/js/saiku/render/SaikuTableRenderer.js

```diff
@@ -351,7 +351,7 @@ SaikuTableRenderer.prototype.internalRender = function(allData, options) {
                         rowContent += '<th class="col_null">&nbsp;</th>';
                     } else {
                         var headerValue = header.value;
-                        if (header.value === 'Yes' || header.value === 'No') {
+                        if (header.value === 'Yes' || header.value === 'No' || header.value === 'N/A') {
```

Comment:
That solution does not account for all cases. There are many other cases when we have answer Yes/No and other options. Try this query: `SELECT DISTINCT LABEL FROM CATEGORY WHERE DTYPE IN (SELECT DTYPE FROM CATEGORY WHERE LABEL = 'Yes' OR LABEL = 'No');`